CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+* June 13, 2018
+
+SYNTAX
```

Comment:
No need in the changelog entry for such changes. Let 's spare the time of the people who read it.